verifier/trustpolicy/oci.go

```diff
@@ -53,41 +53,31 @@ type OCITrustPolicy struct {
 }
 
 // Document represents a trustPolicy.json document
+//
 // Deprecated: Document exists for historical compatibility and should not be used.
 // To create OCI Document, use OCIDocument.
```

Comment:
Should we remove the comment?

Reply:
I was thinking about it as well. However, this file is called `oci.go`, and its changes are all related to `OCI` instead of `blob`. I kind of think it's a good refactoring, it makes the trust policy more extensible. Therefore, I kept it.

Reply:
I see. However, we are still using the deprecated struct `trustpolicy.TrustPolicy` in the test file. Maybe we should fix that in the stable release.

Reply:
That's a good point, we should fix that in the `rc` version. Actually, I think we could keep the `trustpolicy.TrustPolicy` as undeprecated and remove the `OCITrustPolicy` for v1.2.0, meanwhile keeping the `oci.go` and `trustpolicy.go` files as they are good refactoring to the `trustpolicy` package.

Reply:
Updated.